#!/usr/bin/python3
def raise_exception():
    raise TypeError("this is a deliberate typeerror")
# uncomment the foolowing line if you want to test the function directly
#try:
# raise_exception()
# except TypeError as te:
#   print("Exception raised")   
